ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -20,22 +16,28 @@ import {
 import {
   hideGasButtonGroup,
   updateSendErrors,
-} from '../../../../ducks/send/send.duck';
+  setGasLimit,
+  setGasPrice,
+  updateSendAmount,
+  getTokenBalance,
+  getSendMaxModeState,
+  getSendToken,
+  getGasLimit,
+  getGasPrice,
+  getSendAmount,
+} from '../../../../ducks/send';
 import {
   conversionRateSelector as getConversionRate,
   getCurrentCurrency,
   getCurrentEthBalance,
   getIsMainnet,
-  getSendToken,
   getPreferences,
   getBasicGasEstimateLoadingStatus,
   getCustomGasLimit,
   getCustomGasPrice,
```

Comment:
@brad-decker sorry if I missed your answer here. I think you may have said something about how custom-gas is getting changed or removed, but if that's not the case why wouldn't you include it in the migration to the send slice?

Reply:
Or are you think we ought to migrate the whole gas.duck into the send duck at once?

Reply:
the gas slice is used outside of the send slice, in all confirmations that allow editing gas price. I do think it can be greatly/significantly simplified and moved into a controller.